backend/job_descriptions/prompts.py

```diff
@@ -1,4 +1,53 @@
-JOB_DESCRIPTION_PROMPT = (
+# This module contains the prompt templates for generating job descriptions.
+# Main job description template
+JOB_DESCRIPTION_TEMPLATE = """
```

Comment:
This is more of a prompt than a template.

```suggestion
JOB_DESCRIPTION_PROMPT = """
```